cpanfile

```diff
@@ -0,0 +1,22 @@
+requires "Exporter";
+requires "Pod::Checker";
+requires "Pod::Coverage";
+requires "Test::More";
+requires "Test::Pod::Coverage";
+requires "constant";
+
+on 'test' => sub {
+    requires "File::Spec";
+    requires "IO::Handle";
+    requires "IPC::Open3";
+    requires "Test::More";
+    requires "perl", ">= 5.006";
```

Comment:
We probably don't need to require such an old perl, and more to the point, probably should go as a `runtime_requires`, not on `test_requires`.

Reply:
I completely removed it. I copied it from another module and I didn't know if it better to remove it or keep it. thanks.